test/Functional/cypress-commands/README.md

```diff
@@ -0,0 +1,95 @@
+# Cypress OrchardCore
```

Comment:
These docs should be copied to OC docs eventually. But I still need them on NPM as well.